src/core/Global.ttslua

```diff
@@ -353,13 +380,19 @@ end
 -- chaos token stat tracker
 ---------------------------------------------------------
 
-function trackChaosToken(tokenName, matGUID)
+function trackChaosToken(tokenName, matGUID, addOrSubtract)
```

Comment:
Imo a boolean variable would be better for `addOrSubtract`. Could call it `subtract` and then default to adding if it is false or nil.